runtimes/rescript/README.md

```diff
@@ -0,0 +1,30 @@
+# [`@catala-lang/rescript-catala`](https://www.npmjs.com/package/@catala-lang/rescript-catala) ![npm][npm]
+
+A ReScript wrapper for the Catala runtime
+
+## Install
+
+```
+yarn add @catala-lang/rescript-catala
+```
+
+## Local dev
+
+```
+# in this dir
+yarn link
+
+# in the project to use the lib
+yarn link @catala-lang/rescript-catala
+```
+
+> _Somehow it doesn't work with
```

Comment:
?

Reply:
I didn't succeed to have the local package working correctly and I didn't want to loose time on it, as it should not require repetitive updates.